learnbase/src/main/webapp/recommendations.html

```diff
@@ -45,3 +45,4 @@ <h2>Selected Topics</h2>
 </body>
 
 </html>
+
```

Comment:
line 20: only 2 space indent
line 29-31: put `</table>` on line 29 and remove whitespace